File: eCar/eCar.Model/Model/Statistics.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eCar.Model.Model
{
    public class Statistics
    {
        public int Id { get; set; }

        public int? NumberOfHours { get; set; }

        public int? NumberOfClients { get; set; }

        public decimal? PriceAmount { get; set; }

        public DateTime? BeginningOfWork { get; set; }

        public DateTime? EndOfWork { get; set; }

        public int DriverId { get; set; }

        public virtual Driver Driver { get; set; } = null!;
    }
}

File: eCar/UI/ecar_admin/lib/models/Statistics/statistics.dart
import 'package:ecar_admin/models/Driver/driver.dart';
import 'package:json_annotation/json_annotation.dart';

part 'statistics.g.dart';

@JsonSerializable()
class Statistics {
  int? id;
  int? numberOfHours;
  int? numberOfClients;
  double? priceAmount;
  DateTime? beginningOfWork;
  DateTime? endOfWork;
  int? driverId;
  Driver? driver;

  Statistics(
      {this.id,
      this.numberOfHours,
      this.numberOfClients,
      this.priceAmount,
      this.beginningOfWork,
      this.endOfWork,
      this.driverId,
      this.driver});

  factory Statistics.fromJson(Map<String, dynamic> json) =>
      _$StatisticsFromJson(json);

  Map<String, dynamic> toJson() => _$StatisticsToJson(this);
}
